modules/mobile/app/models/mobile/v2/adapters/appointments.rb

```diff
@@ -0,0 +1,328 @@
+# frozen_string_literal: true
+
+module Mobile
```

Comment:
I think somewhere in this file we should add a block comment explaining that this is adapting the VAOS v2 schema into the mobile V0 schema. I agree with your comment that it's not clear that this should really be called v2 because it's not clear if the versions refer to "iterations of appointments adapters" or to the underlying schema. TBH, I think it might be better for it to be tied to the schema, which would allow us to have a v2 adapter for the v2 schema if we ever need one. Maybe we could rename this something like /adapters/v0/v2_appointments` or something. It's not very graceful, but it might be more consistent.

Reply:
what do you think about `/adapters/v0/vaos_v2_appointments` to make more obvious that v2 refers to the upstream data, not the schema?

Reply:
I'd suggest getting a discussion going in the api team room. I'd be curious what others think because I'm still trying to decide what I think.